src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
 }
+
+ul {
```

Comment:
Avoid using tag selectors like `ul`. Use class selectors instead to improve maintainability.